Library/Library/ViewModels/RequestViewModels/AuthorRequestViewModel.cs

```diff
@@ -15,7 +15,10 @@ public class AuthorRequestViewModel
         [Required]
         public string FirstName { get; set; }
 
-        public DateTime? BirthDate { get; set; }
+        [Required(ErrorMessage = "BirthDate cannot be empty")]
```

Comment:
`[Required(ErrorMessage = "{0} cannot be empty")]`
https://stackoverflow.com/questions/3557953/how-to-put-displayname-on-errormessage-format